app/posts/register-trainee-teachers/2020-11-27-beta-prototype-research-round-3.md

```diff
@@ -0,0 +1,169 @@
+  - text: Research playback
+    href: https://docs.google.com/presentation/d/1t5uAjPsI_Y6cnHO9o5__XFfQeMzdX9o54Gz1Yy7a3Ew/
+  - text: User needs
+    href: https://trello.com/b/5WZeE54R/register-trainee-teachers-user-needs-board
+---
+
+Research conducted: 23 to 27 November 2020
+
+The objective of our third research round was to achieve confidence in the proposed ITT data input for the assessment only (AO) route while continuing to improve our provider-led (Pl) journey.
+
+Since UR round 2 we made some key changes to the prototype:
+
+* added the sign in journey
+* changed the process of inputting programme data from manually typing the data to [selecting courses pulled from Publish](/register-trainee-teachers/picking-publish-courses/)
+* changed the [process of recording the assessment outcomes](/register-trainee-teachers/recording-training-outcomes/) to include failure and course extension
+
+The intention was to answer the following questions:
+
+* How do users that are responsible for multiple providers (e.g. lead schools) manage data? 
+* What are the habits and needs around early data input and submitting for TRN (prior to course start)
+* How do providers refer to their courses and how familiar are they with the Publish course codes?
+* Do users understand our review and confirm features, as well as our error messages?
+* Are we meeting user needs around navigation, specifically searching for and sorting  trainee records?
+* Are we meeting user needs around recording assessment outcomes, especially alternative & negative ones?
+
+## Who we tested with
+
+* South West Teacher Training SCITT
+* Inspiring Teacher Training SCITT
+* Lincolnshire Teaching School Alliance SCITT
+* North Manchester ITT Partnership
+* The Tommy Flowers SCITT
+
+## Research findings
+
+### Sign in
+
+There's an immediate brand recognition of BAT and the request to cross-link between the services (e.g. create a landing page/hub at sign in from which they can get into Manage, Publish or Register). This coherence adds to our users’ confidence in our service.
+
+>“I like the continuity between the systems. At the moment all the systems I’m using are completely different to each other, they’re asking for different information. Whereas this is using all the same terms, all the same language, it’s a lot easier to find things. Even down to how it’s structured and formatted on the page it’s easier to see what you’re looking for. Definitely prefer this once it’s all set up.”
+
+>“Would be good if you could get to the services like Manage, Publish and Register more quickly, for example through Google. There isn’t a landing page for these services. In a perfect world you’d have Apply, Publish and this all available on there.”
+
+### Early drafts and requesting TRN
+
+Currently the process of “registering trainees” seems to be viewed as one activity that starts with uploading data onto DTTP and finishes with putting the record through for TRN as a final step.
```

Comment:
Why is 'registering trainees' in quotes?

Reply:
🤷‍♂️ Updated.